PocketKit/Sources/PocketKit/MyList/CompactMyListContainerCoordinator.swift

```diff
@@ -74,13 +70,19 @@ class CompactMyListContainerCoordinator: NSObject {
             self?.present(alert: alert)
         }.store(in: &subscriptions)
 
-        model.archivedItemsList.$presentedWebReaderURL.receive(on: DispatchQueue.main).sink { [weak self] url in
-            self?.present(url: url)
-        }.store(in: &subscriptions)
-
         isResetting = false
         navigationController.delegate = self
     }
+    
+    private func navigate(selectedItem: SelectedItem?) {
+        guard let item = selectedItem else { return }
+        switch item {
+        case .readable(let readable):
+            self.push(savedItem: readable)
+        case .webView(let url):
+            self.present(url: url)
+        }
+    }
```

Comment:
this logic is essentially in two coordinators. Just a thought for the future, would it be helpful to have a `BaseCoordinator` in case we have shared functionality for the coordinators and can remove duplication?